org.jacoco.tests/pom.xml

```diff
@@ -31,8 +31,9 @@
     <module>../org.jacoco.agent.rt.test</module>
     <module>../org.jacoco.agent.test</module>
     <module>../org.jacoco.ant.test</module>
-    <module>../jacoco-maven-plugin.test</module>
+    <module>../org.jacoco.cli.test</module>
     <module>../org.jacoco.examples.test</module>
+    <module>../jacoco-maven-plugin.test</module>
```

Comment:
@Godin Just to have the same order as in `org.jacoco.build`. But I can revert that if you prefer the other sequence.